Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1663 @@
+            file_reputation_action (str | None): File reputation action.
+            anti_malware_scanning_status (str | None): Anti-malware scanning status.
+            suspect_user_agent_scanning (str | None): Suspect user agent scanning.
+            block_malware_categories (List[str] | None): Malware categories to block.
+            block_other_categories (List[str] | None): Other categories to block.
+            settings_status (str): Application settings status.
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        data = remove_empty_elements(
+            {
+                "access_policies": [
+                    {
+                        "policy_name": policy_name,
+                        "amw_reputation": {
+                            "web_reputation": {"filtering": web_reputation_status},
+                            "adv_malware_protection": {
+                                "file_reputation_filtering": file_reputation_filtering_status,
+                                "file_reputation": {
+                                    file_reputation_action: [
+                                        "Known Malicious and High-Risk Files"
+                                    ]
+                                } if file_reputation_action else {},
+                            },
+                            "cisco_dvs_amw": {
+                                "amw_scanning": {
+                                    "amw_scan_status": anti_malware_scanning_status
+                                },
+                                "suspect_user_agent_scanning": suspect_user_agent_scanning,
+                                "block_malware_categories": block_malware_categories,
+                                "block_other_categories": block_other_categories,
+                            },
+                            "state": settings_status
+                        } if settings_status == "custom" else {"state": settings_status},
+                    }
+                ]
+            }
+        )
+        return self._http_request(
+            "PUT",
+            f"{V3_PREFIX}/web_security/access_policies",
+            json_data=data,
+            resp_type="response",
+            ok_codes=[HTTPStatus.NO_CONTENT],
+        )
+
+    def access_policy_delete(self, policy_names: str) -> Response:
+        """
+        Delete access policy.
+
+        Args:
+            policy_names (str): Comma separated policy names to delete.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        params = assign_params(policy_names=",".join(policy_names))
+        res = self._http_request(
+            "DELETE",
+            f"{V3_PREFIX}/web_security/access_policies",
+            params=params,
+            resp_type="response",
+            ok_codes=[HTTPStatus.NO_CONTENT, HTTPStatus.MULTI_STATUS],
+        )
+        return res
```

Comment:
Why do you keep the response in 'res'? 
If there is no special reason, you can just return the `self._http_request`. 👩🏼‍💻